todo_app/data/trello_items.py

```diff
@@ -0,0 +1,97 @@
+import os
+import requests
+import ssl
+from todo_app.data.item import Item
+from requests_kerberos import HTTPKerberosAuth, OPTIONAL
+context = ssl.create_default_context()
+der_certs = context.get_ca_certs(binary_form=True)
+pem_certs = [ssl.DER_cert_to_PEM_cert(der) for der in der_certs]
+with open('wincacerts.pem', 'w') as outfile:
+   for pem in pem_certs:
+      outfile.write(pem + '\n')
+
+def get_items():
+    """
+    Fetches all saved items from the session.
+
+    Returns:
+        list: The list of saved items.
+    """
+    list_items = []
+    api_key = os.getenv("TRELLO_API_KEY")
+    api_token = os.getenv("TRELLO_API_TOKEN")
+    api_list = os.getenv("TRELLO_API_LIST")
+    idBoards = os.getenv("TRELLO_API_BOARD")
+    proxies = {'http':os.getenv("PROXY_URL"),'https':os.getenv("PROXY_URL")}
+    api_url = f'https://api.trello.com/1/boards/{idBoards}/lists'
+    query_params = {
+        "key":api_key,
+        "token":api_token,
+        "cards":"open"
+    }
+    response = requests.get(api_url, params=query_params, proxies=proxies, verify='wincacerts.pem')
+    response_list = response.json()
+
+    for trello_list in response_list:
+        for card in trello_list['cards']:
+            item = Item.from_trello_card(card, trello_list)
+            list_items.append(item)
+    return list_items
+
+
+def add_item(title):
+    """
+    Adds a new item with the specified title to the session.
+
+    Args:
+        title: The title of the item.
+
+    Returns:
+        True if the the item saves.
+    """
+    api_key = os.getenv("TRELLO_API_KEY")
+    api_token = os.getenv("TRELLO_API_TOKEN")
+    api_list = os.getenv("TRELLO_API_OPEN_LIST")
+    proxies = {'http':os.getenv("PROXY_URL"),'https':os.getenv("PROXY_URL")}
+    api_url = "https://api.trello.com/1/cards"
+    query_params = {
+        "key":api_key,
+        "token":api_token,
+        "idList":api_list,
+        "name":title
+        }
+    response = requests.post(api_url,params=query_params, proxies=proxies, verify='wincacerts.pem')
+
+    if response.status_code == '200':
+        return True
+    else:
+        return False
```

Comment:
One of those things you just need to be told exists, but there's actually an `ok` property on the response, which means if Trello sent a success response that wasn't 200 (e.g. a 201) it would still report success, and note that any time we're evaluating a boolean and returning True/False we could just return the boolean directly!
```suggestion
    return response.ok
```